api/Window.json

```diff
@@ -2126,7 +2126,7 @@
               "version_added": null
             },
             "webview_android": {
-              "version_added": null
+              "version_added": false
```

Comment:
This is null for all other Chromium-based browsers, and false for the rest. Seems like some work looking into this entry is needed, can you deal with it separately to simplify this PR?